swagger.yaml

```diff
@@ -0,0 +1,515 @@
+          schema:
+            $ref: "#/definitions/BillingAccountResponse"
+        '422':
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/ErrorModel"
+
+  /billingaccounts/{billingAccountId}:
+    get:
+      tags:
+        - billingaccount
+      security:
+        - Bearer: []
+      description: Retrieve billing account by id, Admin user only.
+      responses:
+        '403':
+          description: No permission or wrong token
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '404':
+          description: Not found
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '200':
+          description: a billing account
+          schema:
+            $ref: "#/definitions/BillingAccountResponse"
+      parameters:
+        - $ref: "#/parameters/billingAcountIdParam"
+      operationId: getBillingAccount
+
+    patch:
+      tags:
+        - billingaccount
+      security:
+        - Bearer: []
+      operationId: updateBillingAccount
+      description: Update a billing account, Admin user only.
+      responses:
+        '403':
+          description: No permission or wrong token
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '404':
+          description: Not found
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '200':
+          description: Successfully updated billing account. Returns original and updated billing account object
+          schema:
+            $ref: "#/definitions/UpdateBillingAccountResponse"
+        '422':
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        default:
+          description: error payload
+          schema:
+            $ref: '#/definitions/ErrorModel'
+      parameters:
+        - $ref: "#/parameters/billingAcountIdParam"
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/BillingAccountBodyParam"
+
+  /billingaccounts/{billingAccountId}/users:
+    get:
+      tags:
+        - billingaccount
+      security:
+        - Bearer: []
+      description: Retrieve users of billing account, Admin user only.
+      operationId: getBillingAccountUsers
+      responses:
+        '403':
+          description: No permission or wrong token
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '404':
+          description: Not found
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '200':
+          description: users of billing account
+          schema:
+            $ref: "#/definitions/BillingAccountUserListResponse"
+      parameters:
+        - $ref: "#/parameters/billingAcountIdParam"
+
+    post:
+      tags:
+        - billingaccount
+      security:
+        - Bearer: []
+      description: Assign user to billing account, Admin user only.
```

Comment:
That was my thought for now.  Getting access to a billing account gives you spending power, so I want to be careful about opening that up any wider.  Currently, only admins in sfdc can add/remove people from a billing account...so it's a bit of gatekeeping going on...